### PR TITLE
Opencti filter update

### PR DESCRIPTION
**Pull request by noydavidi**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-dc.paloaltonetworks.com/browse/CIAC-9809

## Description
A few sentences describing the overall goals of the pull request's commits.

1. docker image changed
2. filters object in the get_indicators function changed according to the new version of OpenCTI
3. not inserting parameters to the command opencti_get_indicators created a bug - fixed by changing the default in this case to filter "ALL"
4. by those changes the "test-module" button is fixed. 
5. bug fix: inserting just score_end or just score_start in the command created an error, now when inserting just one of them, there are default value for each value
6. added some test for those case mentioned above

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/OpenCTI/ReleaseNotes/1_0_11.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+
+#### Integrations
+
+##### OpenCTI
+
+- Updated the Docker image to: *demisto/vendors-sdk:1.0.0.92984*.
+- filters object in the get_indicators function changed according to the new version of OpenCTI
+not inserting parameters to the command opencti_get_indicators created a bug - fixed by changing the default in this case to filter "ALL"
+by those changes the "test-module" button is fixed.
+bug fix: inserting just score_end or just score_start in the command created an error, now when inserting just one of them, there are default value for each value. added some tests for those cases mentioned above
```
Comment (JudahSchwartz) [inline Packs/OpenCTI/ReleaseNotes/1_0_11.md:10]:
```suggestion
Added default values for **score_end** and **just** score_start in the **___** command.
```

File: Packs/OpenCTI/ReleaseNotes/1_0_11.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+
+#### Integrations
+
+##### OpenCTI
+
+- Updated the Docker image to: *demisto/vendors-sdk:1.0.0.92984*.
+- filters object in the get_indicators function changed according to the new version of OpenCTI
```
Comment (JudahSchwartz) [inline Packs/OpenCTI/ReleaseNotes/1_0_11.md:7]: Too much details for release notes. Main point should be "Switched x command to work with version x.y of opencti)

**Review by JudahSchwartz — COMMENTED** [review]
amazing first pr!!!

File: Packs/OpenCTI/Integrations/OpenCTI/OpenCTI.py [hunk 3]
```diff
@@ -107,14 +107,21 @@ def get_indicators(client: OpenCTIApiClient, indicator_types: list[str], score=N
...
-        'key': 'entity_type',
-        'values': indicator_type
-    }]
+    filters: dict[str, Any] = {
+        'mode': 'and',
+        'filters': [{
+            'key': 'entity_type',
+            'values': indicator_type,
+            'operator': 'eq',
+            'mode': 'or'
```
Comment (JudahSchwartz) [inline Packs/OpenCTI/Integrations/OpenCTI/OpenCTI.py:116]: are we sure this should be "or" ?

**Comment by ShirleyDenkberg** [comment]
@noydavidi I agree with Judah's comments.
@JudahSchwartz Doc review completed.

**Review by JudahSchwartz — APPROVED** [review]
🥳

File: Packs/OpenCTI/Integrations/OpenCTI/README.md [hunk 4]
```diff
@@ -1,7 +1,5 @@
 Manages indicators from OpenCTI.  
-This integration is compatible with OpenCTI versions from 4.X to 5.11.X.  
-
-**Note**: Due to [breaking changes to the OpenCTI API on version 5.12.0](https://github.com/OpenCTI-Platform/opencti/releases/tag/5.12.0), this integration is not currently compatible with OpenCTI versions 5.12.0 and above.
+This integration is compatible with OpenCTI versions from 4.X to 5.12.X.
```
Comment (JudahSchwartz) [inline Packs/OpenCTI/Integrations/OpenCTI/README.md:2]: is this still true? compatible version 4.x? didnt we just break it?

**Review by JudahSchwartz — CHANGES_REQUESTED** [review]

**Review by JudahSchwartz — APPROVED** [review]

**Pull request merged**